harness/determined/deploy/local/cluster_utils.py

```diff
@@ -335,7 +373,18 @@ def agent_up(
     )
 
 
-def _kill_containers(containers: docker.models.containers.Container) -> None:
+def _kill_container(container_name: str) -> None:
+    try:
+        container = docker_client.containers.get(container_name)
+        print(f"Stopping {container.name}")
+        container.stop(timeout=20)
+        print(f"Removing {container.name}")
+        container.remove()
+    except docker.errors.NotFound:
+        return
+
+
+def _kill_containers(containers: List[Container]) -> None:
```

Comment:
why do you have `_kill_containers` and `_kill_container`, and they both have the same stop/remove logic encoded into them?

Probably some refactoring would make sense.